v2/pkg/engine/datasource/graphql_datasource/graphql_tws_handler.go

```diff
@@ -93,18 +171,21 @@ func (h *gqlTWSConnectionHandler) StartBlocking(sub Subscription) {
 			switch messageType {
 			case messageTypePing:
```

Comment:
This logic is duplicated in read message. Can we consolidate it?

Reply:
no :D